src/App.css

```diff
@@ -57,3 +66,48 @@ button:active {
   box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.2);
   transform: scale(0.95);
 }
+
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 5%;
+  margin-inline: 10%;
```

Comment:
[ Optional Change ]

- [ ] For alignment purposes, It would be nice if you could remove the padding here so the nav items stay left and right and then aligned with the page contents nicely, especially for the calculator page. 👍 
- I removed it and it displays nicely as in the last screenshot below. 

![image](https://user-images.githubusercontent.com/50941074/208140080-0b0dffbe-ec56-4069-8e3f-3fe19b4a70f4.png)

![image](https://user-images.githubusercontent.com/50941074/208140427-07006e93-8577-4239-b174-f4924564060b.png)

Reply:
I appreciate your suggestions a lot, sir, seems to me it would be a better-looking section with them.